### DISCUSSION TITLE
Is this project stale?

### DISCUSSION THREAD

**Discussion by SebastianClaesson** [benclaussen/NetboxPS#71] [General]
Is this project stale? Or is it continously developed?

I've started using this as part of a proof of concept and before I start using this anywhere in production it would be nice to know if the project is maintained or if it's stale.

Thanks!

**Comment by alagoutte** [comment]
Hi Sebastian,

It is more or less stale...

It is not my priority for the moment for me (i don't known for @benclaussen...)

I see also the patchs of @tigpas and i think, may be create a dev-v4 release branch and publish pre release ?

I look also other Netbox Project on PS Gallery and i think the more serious project is [netbox-rest-module](https://www.powershellgallery.com/packages/netbox-rest-module/)

May be also look [Auto Rest Generator](https://github.com/Azure/autorest.powershell?tab=readme-ov-file) for automatically generate module from OpenAPI Json
